2-mandatory/3-financial-times.js

```diff
@@ -5,33 +5,58 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const out = [];
+    for (z in allArticleTitles) {
```

Comment:
for-in loop iterates over object properties. In this case we need to use for-of

Reply:
While this is true, it might be helpful to include an example (maybe even using the actual code in the PR) of how your suggestion would look. You can reply to this comment with how you think the code should look.

Reply:
for (const articleTitle of allArticleTitles) {
    if (articleTitle.length <= 65) {
      out.push(articleTitle);
    }